### DISCUSSION TITLE
0.85s for default cube rendering. how to speed up?

### DISCUSSION THREAD

**Discussion by HSDSZ** [bnpr/Malt#157] [Q&A]
I rendered a default cube. malt used 0.85s.
Can I speed up the render by upgrading my CPU or GPU?

My current computer
CPU: i5-9400F
GPU: GTX1660Ti

**Comment by HSDSZ** [comment]
Rendering a default cube in EEVEE uses less than 0.3s. I am wondering what is slowing my rendering in malt.

**Comment by pragma37** [comment]
Malt is slower than EEVEE in general.
Some performance tips:
https://github.com/bnpr/Malt/discussions/81#discussioncomment-718682
https://bnpr.gitbook.io/beer-malt-wiki/tips/speed-up-malt

On the hardware side, the GPU is what will affect rendering performance the most.
The CPU can become a bottleneck on large scenes or with very heavy meshes.
